package.json

```diff
@@ -24,10 +24,7 @@
     "postinstall": "node ./tools/rebuildNativeModules.js&&webpack",
     "test": "NODE_ENV=test mocha --compilers js:babel-register --recursive $(find test -name '*Test.js')"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/bbondy/electron-sample.git"
-  },
+  "repository": "brave/browser-laptop",
```

Comment:
https://docs.npmjs.com/files/package.json#repository
Is this maybe the preferred format? If so mind doing an amend of this pull request?
Thanks for the PR.